test/runtests.jl

```diff
@@ -36,10 +36,14 @@ const GROUP = get(ENV, "GROUP", "All")
         @safetestset "Quadratically regularized OT" begin
             include("quadratic.jl")
         end
+
+        @safetestset "Gromov-Wasserstein OT" begin
+            include("gromov.jl")
+        end
     end
 
     # CUDA requires Julia >= 1.6
-    if (GROUP == "All" || GROUP == "GPU") && VERSION >= v"1.6"
+    if (GROUP == "All" || GROUP == "GPU") && VERSION >= v"1.8"
```

Comment:
Why is 1.8 needed here?

Reply:
Sorry, still in the middle of playing around to see why the buildkite CI fails.